WalletWasabi.Packager/Program.cs

```diff
@@ -622,4 +618,19 @@ private static string GetBuildInfoData()
 
 		return output;
 	}
+
+	private static bool TryStartProcessAndWaitForExit(string command, string workingDirectory, [NotNullWhen(true)] out string? result, string? writeToStandardInput = null, string? arguments = null, bool redirectStandardOutput = false)
+	{
+		result = null;
+		try
+		{
+			result = StartProcessAndWaitForExit(command, workingDirectory, writeToStandardInput, arguments, redirectStandardOutput)?.Trim() ?? "";
+			return true;
+		}
+		catch
+		{
+			// Exceptions already displayed in StartProcessAndWaitForExit.
+		}
```

Comment:
Oh boy, we will have another rant about this empty catch block, and that it might break: Wasabi, Bitcoin, maybe even the whole universe.

Reply:
That's why I added a comment there :)

Reply:
Actually I noticed some path where the message was not written out so I added logging there.